lib/Optimizer/CodeGen/TranslateToOpenQASM.cpp

```diff
@@ -23,13 +23,12 @@ using namespace cudaq;
 //===----------------------------------------------------------------------===//
 // Helper functions
 //===----------------------------------------------------------------------===//
-
```

Comment:
nit: Leave the blank line since the banner comment and the doxygen comment are really doing different things here.

Reply:
This was removed at one point in PR#2401 and restored later. So, aligning with the latest mainline should resolve it.

Reply:
done!